pkg/build/message.go

```diff
@@ -0,0 +1,23 @@
+package build
+
+import (
+	"path/filepath"
+
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+const (
+	messageScriptName = "message.sh"
+)
+
+func ConfigureMessage(buildConfig *config.BuildConfig) error {
```

Comment:
We should opt for using unexported functions if we don't expect them to be called outside of a particular package.

Reply:
This was another place I was going to specifically ask you about. I'll change.